openseadragon-scalebar.js

```diff
@@ -418,6 +418,21 @@
             return getScalebarSizeAndText(ppmi, minSize, "mi");
         },
         /**
+         * Astronomy units. Choosing the best unit from arcsec, arcminute, and degree
+         */
+        ASTRONOMY: function(ppa, minSize) {
+	    var maxSize = minSize * 2;
```

Comment:
Right, it looks good here: https://github.com/youngmd/OpenSeadragonScalebar/blob/ce5e0abbd8d4b5a80be8edbdfc8c7078801245fc/openseadragon-scalebar.js
Must be a github bug.